test/integration/integration_test_fixtures.py

```diff
@@ -84,15 +90,22 @@ def setUp(self):
     """
     Performs sanity checks and starts up the server. Upon exit the server is ready to accept connections.
     """
-    assert (os.path.exists(self.nighthawk_test_server_path))
-    assert (os.path.exists(self.nighthawk_client_path))
-    test_id = os.environ.get('PYTEST_CURRENT_TEST').split(':')[-1].split(' ')[0].replace(
-        "[", "_").replace("]", "")
-    self.parameters["test_id"] = test_id
+    if os.getenv("NH_DOCKER_IMAGE", "") == "":
+      assert (os.path.exists(self.nighthawk_test_server_path))
```

Comment:
Can we amend these assertions with some error message that would help the receiver of this error message understand what wen't wrong?

Something like:
assert False, "custom error message"